index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="header">
```

Comment:
[OPTIONAL]

Excellent work implementing changes requested by the previous code reviewer! Just to add the cherry on top, you could remove the class and select this element by its `<header>` tag :cherries: